04.ls/ls.rb

```diff
@@ -4,7 +4,7 @@
 require 'optparse'
 require_relative 'lib/ls_command'
 
-option = ARGV.getopts('a', 'r', symbolize_names: true)
+option = ARGV.getopts('a', 'r', 'l', symbolize_names: true)
```

Comment:
```suggestion
options = ARGV.getopts('a', 'r', 'l', symbolize_names: true)
```

今さらですが複数形の方がいいかも。
getoptsもget optionsの略でしょうし。